server/test/unit/converters/schema/test_remix.py

```diff
@@ -0,0 +1,219 @@
+import json
+import os
+import unittest
+import unittest.mock
+
+import anndata
+import numpy
+import pandas as pd
+import scanpy as sc
+
+from server.converters.schema import remix
+
+PROJECT_ROOT = os.popen("git rev-parse --show-toplevel").read().strip()
+
```

Comment:
Is it possible to have one E2E test for the `apply_schema` function here?

Reply:
Or something involving the config file? Might be nice to have a simple sample.

Reply:
Yep, added